controller_manager/src/controller_manager.cpp

```diff
@@ -2466,6 +2486,26 @@ controller_interface::return_type ControllerManager::update(
       RCLCPP_ERROR(
         get_logger(), "Activating fallback controllers : [ %s]", controllers_string.c_str());
     }
+    std::vector<std::string> failed_controller_interfaces, fallback_controller_interfaces;
+    failed_controller_interfaces.reserve(500);
```

Comment:
Isn't this in the update method? Can we do this ahead of time?

Reply:
@destogl I was handling it in a different PR: https://github.com/ros-controls/ros2_control/pull/1801